packages/Babylonian-Tests.package/BPPrintbuggerTest.class/instance/tearDown.st

```diff
@@ -0,0 +1,4 @@
+running
+tearDown
+	
+	BPExamples removeAll
```

Comment:
This should be `BPExamples value`. It is not too bad as removeAll uses `#value` itself but it looks scary at first.